### PR TITLE
Improve verify builds tab

### PR DESCRIPTION
**Pull request by niclimcy**

### CODE REVIEW

File: src/components/verify-tab/VerifyTabPage.vue [hunk 1]
```diff
@@ -5,70 +5,83 @@
...
-        This page lets you verify whether the OTA package you downloaded is signed with a LineageOS
-        private key and hasn't been tampered with.<br />
-        NOTE: This only works for OTA packages; it won't work with boot.img, recovery.img, etc.
+    <div class="list-container" data-simplebar>
+      <div class="list">
+        <div class="header">
+          <h1>Verify builds</h1>
+          <div>
+            <p>
+              Verify downloaded builds are signed with LineageOS private keys and are untampered by
```
Comment (luk1337) [inline src/components/verify-tab/VerifyTabPage.vue]: not sure if I like the way this sentence starts.

**Comment by luk1337** [comment]
I removed navbar change and also replaced `<br />` with something slightly better (see compare for diff).

File: src/components/verify-tab/VerifyTabPage.vue (hunk 1 above)
Comment (niclimcy) [reply src/components/verify-tab/VerifyTabPage.vue]: "You can verify that downloaded builds..."?

Comment (luk1337) [reply src/components/verify-tab/VerifyTabPage.vue]: maybe!

**Comment by luk1337** [comment]
I preferred old title + languagetool suggested adding "that", so I did so.

**Pull request merged**